unionml/model.py

```diff
@@ -589,17 +592,19 @@ def remote(
         :param registry: Docker registry used to push UnionML app.
         :param image_name: image name to give to the Docker image associated with the UnionML app.
         :param dockerfile: path to the Dockerfile used to package the UnionML app.
+        :param docker_install_path: path where the UnionML source is installed within the docker image.
```

Comment:
a more intuitive name where would be `patch_destination_dir`, to link this to the `patch` argument.